mappings/net/minecraft/world/biome/layer/layerUtilities/InitLayer.mapping

```diff
@@ -1,6 +1,7 @@
-CLASS chl net/minecraft/world/biome/layer/InitLayer
+CLASS chl net/minecraft/world/biome/layer/layerUtilities/InitLayer
```

Comment:
`layerUtilities` is a bit long. Maybe just `util(s)`? Also, I think package names should be lowercase.

Reply:
for me they need to be distinguish as sampling and layer utilities are basically useless when reading that part of the code (we all know the numbers of the samplers and the layer are just the structure) Seed utility are the PRNG tempering for chunkX chunkZ